Set up 'Getting started' page

fsteeg opened the pull request:
Will resolve #19.

TobiasNx commented on getting-started.html:
```diff
@@ -0,0 +1,57 @@
+<p>To set up IDE support for editing your Flux and Fix files, see <a href="/ide-extensions/index.html">the IDE extensions page</a>.</p>
+<h2 id="next-steps">Next steps</h2>
+<p>For a complete introduction to Metafacture in German, check out the latest iteration of <a href="https://slides.lobid.org/2022-12-metafacture-workshop/#/">our Metafacture Workshop</a>. For different use cases, e.g. using Metafacture as a library, using the Morph language, and more, see <a href="https://github.com/metafacture/metafacture-documentation">our documentation collection</a>.</p>
+</body>
```
Should we link a guide for the Java Integration too?

fsteeg commented on getting-started.md:
```diff
@@ -0,0 +1,28 @@
+
+## Playground
+
+The easiest way to get started with Metafacture is the Playground. Take a look at the [first example](https://metafacture.org/playground/?flux=PG_DATA%0A%7Cas-lines%0A%7Cdecode-formeta%0A%7Cfix%0A%7Cencode-xml%28rootTag%3D%22collection%22%29%0A%7Cprint%0A%3B&fix=move_field%28_id%2C+id%29%0Amove_field%28a%2C+title%29%0Apaste%28author%2C+b.v%2C+b.n%2C+%27~aus%27%2C+c%29%0Aretain%28id%2C+title%2C+au ...
```
Sounds good!

TobiasNx commented:
@fsteeg process fails

fsteeg commented:
Updated the action to use the `ubuntu-latest` runner in https://github.com/metafacture/metafacture.github.io/pull/20/commits/e95dde9382cd277a72cbd6e2da7599a102f0a380 and added a missing update in https://github.com/metafacture/metafacture.github.io/pull/20/commits/75b82bfd1183093905ab1a9bddde30ee690dd438, now it looks good, HTML was committed by the GitHub Action in https://github.com/metafacture/metafacture.github.io/pull/20/commits/09a88de6a06602a3726704e9e5fddb6fec3e7382.
